admin/tools/docker/build-mariadb-image.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+# Create a MariaDB+Scisql docker image.
+# Use scisql master tip
+
+# @author  Fabrice Jammes, IN2P3/SLAC
+
+set -e
+
+DIR=$(cd "$(dirname "$0")"; pwd -P)
+. "$DIR/conf.sh"
+
+MARIADB_VERSION="10.1.25"
```

Comment:
I think this should not be hardcoded in a script. I also see the same number in mariadb/Dockerfile, how do keep them consistent?